dhis-2/dhis-services/dhis-service-core/src/main/java/org/hisp/dhis/dataelement/DefaultDataElementCategoryService.java

```diff
@@ -508,6 +511,9 @@ public void generateDefaultDimension()
         category.addCategoryOption( categoryOption );
         addDataElementCategory( category );
 
+        category.setPublicAccess( AccessStringHelper.CATEGORY_DEFAULT );
+        updateDataElementCategory( category );
+

```

Comment:
Category is not enable for data sharing. Check CategorySchemaDescriptor, it doesnt have "schema.setDataShareable( true );"